libs/remix-ui/run-tab/src/lib/components/account.tsx

```diff
@@ -386,16 +387,25 @@ export function AccountUI(props: AccountProps) {
       <label className="udapp_settingsLabel">
         <FormattedMessage id="udapp.account" />
         {!smartAccountSelected ? <CustomTooltip placement={'top'} tooltipClasses="text-wrap" tooltipId="remixPlusWrapperTooltip" tooltipText={plusOpt.title}>
-          <span id="remixRunPlusWrapper">
-            <i id="remixRunPlus" className={`ml-2 fas fa-plus udapp_icon ${plusOpt.classList}`} aria-hidden="true" onClick={newAccount}></i>
+          <span className="px-1" id="remixRunPlusWrapper">
+            <i id="remixRunPlus" className={`fas fa-plus udapp_icon ${plusOpt.classList}`} aria-hidden="true" onClick={newAccount}></i>
           </span>
         </CustomTooltip> : null }
         {!smartAccountSelected ? <CustomTooltip placement={'top'} tooltipClasses="text-nowrap" tooltipId="remixSignMsgTooltip" tooltipText={<FormattedMessage id="udapp.signMsgUsingAccount" />}>
-          <i id="remixRunSignMsg" data-id="settingsRemixRunSignMsg" className="ml-2 fas fa-edit udapp_icon" aria-hidden="true" onClick={signMessage}></i>
+          <i id="remixRunSignMsg" data-id="settingsRemixRunSignMsg" className="mx-1 fas fa-edit udapp_icon" aria-hidden="true" onClick={signMessage}></i>
         </CustomTooltip> : null }
-        <span >
-          <CopyToClipboard className="fas fa-copy ml-2 p-0" tip={intl.formatMessage({ id: 'udapp.copyAccount' })} content={selectedAccount} direction="top" />
-        </span>
+        {!smartAccountSelected ?
```

Comment:
Not sure what is being done here but I do not see that copy or i icon when a smart account is selected
<img width="277" alt="Screenshot 2025-05-20 at 1 08 28 PM" src="https://github.com/user-attachments/assets/a3578eaf-c15a-4be7-83e9-9bb6cd48258b" />

Reply:
the I is about Autorization. copy had a bug I fixed but didn;t know you need it there, sorry I'll update it